dist/tools/nacl_key_generator/README.md

```diff
@@ -0,0 +1,13 @@
+# Key-pair generator using [NaCL](https://nacl.cr.yp.to/)
+This program will generate two key-pairs and store them as binary files, using
+tweetNaCl package in RIOT
```

Comment:
YEs I though a lot about it and didn't find a "clean" solution. What can I do is to check it out using git or something like that, just to avoid the code being part of RIOT.